src-plugins/mscAlgorithmPaint/mscAlgorithmPaintToolBox.cpp

```diff
@@ -399,6 +399,7 @@ AlgorithmPaintToolBox::AlgorithmPaintToolBox(QWidget *parent ) :
 
     m_applyButton = new QPushButton( tr("Save") );
     m_applyButton->setToolTip(tr("Save result to the Database"));
+    m_applyButton->setObjectName("saveButton");
```

Comment:
Yep, almost each time i had to add an object name, it was "fooButton" :). I prefer also \o